pkgs/applications/networking/instant-messengers/viber/unwrapped.nix

```diff
@@ -0,0 +1,155 @@
+{ lib
+, fetchurl
+, makeBinaryWrapper
+, autoPatchelfHook
+, dpkg
+, alsa-lib
+, nss
+, gst_all_1
+, stdenvNoCC
+, xorg
+, lcms
+, libwebp
+, snappy
+, gdk-pixbuf
+, libmng
+, gtk3
+, pango
+, libtiff
+, tslib
+, mtdev
+, libxslt
+, libpulseaudio
+, mesa
+, xcb-util-cursor
+, libxkbcommon
+, libglvnd
+, brotli
+, cups
+, libevent
+, fontconfig
+, freetype
+, libkrb5
+, harfbuzz
+, nspr
+, python311Packages
+, zstd
+, udev
+}:
+
+stdenvNoCC.mkDerivation {
+
+  pname = "viber";
+  version = "21.0.0.1";
+  meta = {
+    homepage = "https://www.viber.com";
+    description = "An instant messaging and Voice over IP (VoIP) app";
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+    license = lib.licenses.unfree;
+    platforms = [ "x86_64-linux" ];
+    maintainers = with lib.maintainers; [ jagajaga liarokapisv ];
+  };
+
+  src = fetchurl {
+    # Official link: https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb
+    url = "https://web.archive.org/web/20231030150355/https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb";
+    sha256 = "sha256-47E6emeIYyG0jid7Oqrn2CEKtb9nQhOckFtEMBevZGA=";
+  };
+
+  # Needed due to failing dlopen calls.
```

Comment:
Can you add just a bit more info like which library?

Reply:
Yes, it was `libudev` as well as `libpci` for the previous version. In the latest only `libudev` is still an issue.

Reply:
(Sandro meant as a comment, in the code.)

Reply:
I know, just commented for those curious.